modules/ROOT/pages/release-notes/release-notes509.adoc

```diff
@@ -20,29 +20,29 @@ NOTE: TinyMCE 5.0.8 community was not released to enterprise due to issues found
 
 TinyMCE 5.0.9 adds back the ability to include multiple toolbars in the editor, similar to TinyMCE 4.x. Multiple toolbars can be configured by using the toolbar array or toolbar1, toolbar2, etc... configuration options.
 
-image::{baseurl}/images/multiple-toolbars.png[Multiple toolbars]
+image::multiple-toolbars.png[Multiple toolbars]
 
-For more information on multiple toolbars refer to the link:{baseurl}/configure/editor-appearance/#usingmultipletoolbars[documentation].
+For more information on multiple toolbars refer to the xref:configure/editor-appearance.adoc#usingmultipletoolbars[documentation].
 
 NOTE: If multiple toolbars are used, the toolbar drawer will be disabled as toolbar drawers aren't compatible with multiple toolbars.
 
 ==== base_url and suffix options
 
 TinyMCE 5.0.9 adds two new configuration options: `base_url` and `suffix`. These new options provide an easier way to provide the editor manager `baseURL` and `suffix` properties.
 
-For more information on the new options refer to the link:{baseurl}/configure/integration-and-setup/#base_url[base_url] and link:{baseurl}/configure/integration-and-setup/#suffix[suffix] documentation.
+For more information on the new options refer to the xref:configure/integration-and-setup.adoc#base_url[base_url] and xref:configure/integration-and-setup.adoc#suffix[suffix] documentation.
 
 ==== Media plugin m4a file support
 
 The *Media* plugin now supports the `M4A` audio format. Inserting a m4a file will now be embedded using a HTML5 `<audio>` element.
 
-For more information on the media plugin refer to the link:{baseurl}/plugins/opensource/media/[documentation].
+For more information on the media plugin refer to the xref:plugins/opensource/media.adoc[documentation].
 
 == Updates and enhancements
 
 === TinyMCE
 
-The new TinyMCE 5.0.9 editor comes with numerous changes to the previous versions. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version509june262019[changelog].
+The new TinyMCE 5.0.9 editor comes with numerous changes to the previous versions. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version509june262019[changelog].
```

Comment:
This xref doesn't resolve as the `version509june262019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version509june262019]]
== 5.0.9 - 2019-06-26
```